# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: .cargo/config.in
# Commit: 57b2cda7e0db
# Full Hash: 57b2cda7e0db8b07fcc1e09113ce26add178c68a
# Author: Sammy Khamis <skhamis@mozilla.com>
# Date: 2023-03-01 03:56:13
# Regressor Bug: 1818349
# File Overlap Count: 23
# Overlapping Files: third_party/rust/sync15/src/client/coll_update.rs, third_party/rust/sync15/src/client/mod.rs, third_party/rust/sync15/.cargo-checksum.json, third_party/rust/sync15/src/client/sync.rs, Cargo.toml
# Description:
#   Bug 1818349 Part 1: Vendor new version of application-services r=markh
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D171181
# ==============================================================================

diff -r eaf9f12a1481 -r 57b2cda7e0db .cargo/config.in
--- a/.cargo/config.in	Tue Feb 28 04:22:35 2023 +0000
+++ b/.cargo/config.in	Tue Feb 28 05:14:29 2023 +0000
@@ -15,11 +15,6 @@
 rev = "a6514854d4518b02f2805719ff6cd74dae7dfde6"
 replace-with = "vendored-sources"
 
-[source."https://github.com/bendk/application-services"]
-git = "https://github.com/bendk/application-services"
-rev = "ecb35df5fc40357c49922f90e86bf4147fa52953"
-replace-with = "vendored-sources"
-
 [source."https://github.com/chris-zen/coremidi.git"]
 git = "https://github.com/chris-zen/coremidi.git"
 rev = "fc68464b5445caf111e41f643a2e69ccce0b4f83"
@@ -80,6 +75,11 @@
 rev = "688a6574cf830b4a3adaf9c5d41efc1f7dd4017f"
 replace-with = "vendored-sources"
 
+[source."https://github.com/mozilla/application-services"]
+git = "https://github.com/mozilla/application-services"
+rev = "5433b3dc9ac152a0da29ed6642df839d79515a59"
+replace-with = "vendored-sources"
+
 [source."https://github.com/mozilla/audioipc"]
 git = "https://github.com/mozilla/audioipc"
 rev = "fb7a2b12ced3b43e6a268621989c6191d1ed7e39"